clients/src/main/java/org/apache/kafka/common/requests/PartitionState.java

```diff
@@ -20,21 +20,24 @@
 
 import java.util.List;
 
+// TODO: rename this class to LeaderAndIsrPartitionInfo
```

Comment:
Sure. I think we can do that refactor. Since this is not an important refactor, can we do this refactor later or even in separate patch to reduce the chance of conflicts with other commits?

For now I replaced the TODO with a comment that says `This is used to describe LeaderAndIsrPartitionInfo`.